### ISSUE TITLE
BatchNorm1D does not work with batches of size 1

### ISSUE THREAD

**Issue by SebieF** [sacdallago/biotrainer#49]
The LightAttention model used for residues_to_class protocol uses BatchNorm1D. However, if using a batch size of 1 is not possible with BatchNorm1D. Because a batch size of 1 is an edge case anyway, we decide not to fix this problem soon. See also #34.

Please answer to this issue if you should encounter the problem, then we might look into a solution for it.

**Comment by SebieF** [comment]
Batches of size 1 seem also to be a problem for the torch.nn.BCEWithLogitsLoss used for protein_protein_interaction protocol.
